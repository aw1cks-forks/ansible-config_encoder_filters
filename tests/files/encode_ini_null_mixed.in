var1: '!!!null'
var2: abc

section1:
  var3: '!!!null'
  var4: def
