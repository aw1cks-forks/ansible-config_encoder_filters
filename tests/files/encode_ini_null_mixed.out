var1
var2=abc

[section1]
var3
var4=def
